views/default/elgg.js.php

```diff
@@ -18,6 +18,8 @@
 // For backwards compatibility...
 echo elgg_view('sprintf.js');
 
+echo elgg_view('elgg/widgets.js');
```

Comment:
Need a comment before this, like "Inlining this is required for the elgg.ui.widgets API to be defined immediately. Without this, 3rd party plugins calling elgg.ui.widget methods will break."